lib/nero/pulsar_hunters/pulsar_hunters_algorithm.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'pulsar_hunters_classification'
+require_relative 'pulsar_hunters_estimate'
+
+module Nero
+  module PulsarHunters
+    class PulsarHuntersAlgorithm < Nero::Algorithm
+      def process(classification, agent, estimate)
```

Comment:
Unused method argument - `agent`. If it's necessary, use `_` or `_agent` as an argument name to indicate that it won't be used.<br>Assignment Branch Condition size for process is too high. [19.52/15]